Install-Django-Flask.md

```diff
@@ -0,0 +1,247 @@
+
+It is cumbersome running long commands like this. Fortunately, there is something like `package.json` in Python domain as well - a list of dependencies, which the package manager can use to duplicate the environment by downloading them with proper version from central repo.
+
+The standard is to use `pip freeze` and log the output to a local file, which can be source controlled.
+
+```sh
+$ pip freeze > requirements.txt
+```
+
+Here's the content of `requirements.txt` from my environment, after installing those Flask packages. You may add or remove more packages as your application grows; but for now, just copy paste the content of the following in a text file in the same directory as you are in.
+
+```
+Babel==2.2.0
+Flask==0.10.1
+Flask-Babel==0.9
+Flask-Login==0.3.2
+Flask-Mail==0.9.1
+Flask-OpenID==1.2.5
+Flask-SQLAlchemy==2.1
+Flask-WTF==0.12
+Flask-WhooshAlchemy==0.56
+Jinja2==2.8
+MarkupSafe==0.23
+SQLAlchemy==1.0.12
+Tempita==0.5.2
+WTForms==2.1
+Werkzeug==0.11.4
+Whoosh==2.7.2
+blinker==1.4
+coverage==4.0.3
+decorator==4.0.9
+defusedxml==0.4.1
+flipflop==1.0
+guess-language==0.2
+itsdangerous==0.24
+pbr==1.8.1
+python3-openid==3.0.9
+pytz==2015.7
+six==1.10.0
+speaklater==1.3
+sqlalchemy-migrate==0.10.0
+sqlparse==0.1.18
+```
+
+This list of packages are taken from [here](http://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-i-hello-world).
+
+Once you have saved the file, just run
+
+```
+$ pip install -r requirements.txt
+```
+
+The package manager would take care of installing missing packages for you! And you should commit this file with your source control system.
+
+The above set of commands assume that you have a Linux machine or Mac OSX machine; or you are using a cloud-hosted box on cloud9 or Nitrous; or maybe you are using a Vagrant box.
+
+But, if you have to use a Windows machine, do consider using Windows Powershell, instead of Windows CMD. Most of the commands would be same. In case you need any assistance, you might want to check [this Stack Overflow discussion](http://stackoverflow.com/questions/17917254/how-to-install-flask-on-windows).
+
+### Wrapping Up
+
+We have successfully installed the web-framework of our need. However, it's not yet complete. Most web applications are content and data driven - so we need a data storage. Or, a Database, if you will.
+
+In next article, we would be discussing how to install PostgreSQL and use it with our Python web application.
+
+A point to ponder - we have been using `pip` heavily, but we have barely said anything about it. Well, for now, it's just a package manager like `npm`. It has some differences with `npm`; but, you don't need to worry about that now. If you are interested, do check out the [official `pip` documentation](http://pip-python3.readthedocs.org/en/latest/index.html).
+
+If you have suggestions or questions, come join us on [gitter](https://gitter.im/FreeCodeCamp/FreeCodeCamp).
```

Comment:
_If you have suggestions or questions, come join us on [gitter](https://gitter.im/FreeCodeCamp/FreeCodeCamp)_.